docs/publications.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Publications for Project Verona
+---
+# Publications
+
+This page contains the papers related to Project Verona. They are grouped by topic.
+
+## Ownership
+
+- [Dynamic Region Ownership for Concurrency Safety](https://www.microsoft.com/en-us/research/publication/dynamic-region-ownership-for-concurrency-safety/)
+  Fridtjof Peer Stoldt, Brandt Bucher, Sylvan Clebsch, Matthew Johnson, Matthew J. Parkinson, Guido Van Rossum, Eric Snow and Tobias Wrigstad, PLDI 2025.
```

Comment:
NIT:

```suggestion
  Fridtjof Stoldt, Brandt Bucher, Sylvan Clebsch, Matthew Johnson, Matthew J. Parkinson, Guido Van Rossum, Eric Snow and Tobias Wrigstad, PLDI 2025.
```

I usually don't include my second name. I've also updated my name on researchr.org to just be "Fridtjof Stoldt"